source/patterns/01-molecules/inline-list/inline-list.twig

```diff
@@ -0,0 +1,17 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['inline-list']) }) -%}
+
+<ol {{- html.attributes(attributes) -}}>
+  {%- for item in items -%}
+
+    {%- with item|merge({html: html}) only -%}
+      {%- set attributes = attributes|default({}) -%}
+      <li {{- html.attributes(attributes) -}}>
```

Comment:
Would be consistent with TagList to allow attributes to be set here and wrap the contents inside `content` https://github.com/libero/pattern-library/blob/084ce687fd091a5998584d46d0424a875aa32a46/source/patterns/01-molecules/tag-list/tag-list.twig#L33-L41

or don't and have the `with` on the inside like https://github.com/libero/pattern-library/blob/a361fd5f9b0a452e2e155604a16732a0b09d7738/source/patterns/02-organisms/content-header/content-header.twig#L55-L59

Not entirely convinced the former is needed, but probably better to be consistent.

Reply:
Like https://github.com/libero/pattern-library/pull/108/commits/b37d9e1e1fd805f16d010f7e28b3b48461f35735?

Reply:
Almost, pushed d0a2cfe.